tests/PhpSpreadsheetTests/Calculation/DateTimeTest.php

```diff
@@ -178,8 +178,19 @@ public function providerWEEKDAY()
      */
     public function testDAY($expectedResult, ...$args)
```

Comment:
you can simplify the method body if you use a declaration like `testDAY($expectedResultExcel, $expectedResultOpenOffice, ...$args)`. So you don't have to use tricks like `array_shift()`

Reply:
done.